qa/L0_perf_nomodel/run_test.sh

```diff
@@ -47,13 +47,17 @@ TRITON_DIR=${TRITON_DIR:="/opt/tritonserver"}
 ARCH=${ARCH:="x86_64"}
 SERVER=${TRITON_DIR}/bin/tritonserver
 BACKEND_DIR=${TRITON_DIR}/backends
-DATADIR=${DATADIR:="/data/inferenceserver/${REPO_VERSION}"}
 MODEL_REPO="${PWD}/models"
 PERF_CLIENT=../clients/perf_client
 TF_VERSION=${TF_VERSION:=2}
 SERVER_ARGS="--model-repository=${MODEL_REPO} --backend-directory=${BACKEND_DIR} --backend-config=tensorflow,version=${TF_VERSION}"
 source ../common/util.sh
 
+# DATADIR is already set in environment variable for aarch64
+if [ "$ARCH" != "aarch64" ]; then
```

Comment:
> because the non-Jetson benchmark tests don't set the DATADIR

I think it's the opposite. I think the x86 tests (infra/CI script) were setting the DATADIR, but to the wrong value for this test.

Then adding the new behavior of "keep datadir if set, otherwise set it to this default" from your last PR kept it as the incorrect value for the test.

To summarize, I think your previous PR's logic of "keep datadir if set, otherwise set it to this test-specific default" is good and removes the need for ARCH checks in the tests. Instead, it's on us to set the right datadir in the CI scripts. 

I'll share some links offline.

Reply:
Actually after looking through it, I think it'll be more trouble than its worth. It looks like we'd basically have to explicitly `unset DATADIR` in the benchmark test template which isn't very intuitive. Probably better to keep the logic in the test script for now.